l10n_it_invoices_data_communication/__manifest__.py

```diff
@@ -16,7 +16,7 @@
     'depends': [
         'account', 'l10n_it_fiscal_document_type', 'l10n_it_codici_carica',
         'l10n_it_fiscalcode', 'l10n_it_esigibilita_iva',
-        'l10n_it_account_tax_kind',
+        'l10n_it_account_tax_kind', 'l10n_it_account',
```

Comment:
Eliminerei la dipendenza diretta a `l10n_it_account` in quanto `l10n_it_fiscal_document_type` dipende già da `l10n_it_account`

Reply:
https://github.com/OCA/l10n-italy/pull/1515#discussion_r345665399